includes/wikia/nirvana/WikiaSkin.class.php

```diff
@@ -283,8 +283,11 @@ public function getStylesWithCombinedSASS(Array &$sassFiles) {
 
 		wfDebug( sprintf( "%s: combined %d SASS files\n", __METHOD__, count($sassFiles) ) );
 
+		$main = strpos($sassFilesUrl, 'ads.scss') !== false ?
```

Comment:
It's worth updating `WikiaSkinTest::testGetStylesWithCombinedSASS` since we changing the method.